app/views/bookmarks/_tools.html.erb

```diff
@@ -1,7 +1,7 @@
-<ul class="<%= controller_name %>Tools nav nav-pills">
+<div class="<%= controller_name %>Tools">
   <%= render_show_doc_actions document_list, document: nil, document_list: @response.documents, url_opts: Blacklight::Parameters.sanitize(params.to_unsafe_h) do |config, inner| %>
-    <li class="nav-item">
+    <button class="btn btn-outline-primary">
       <%= inner %>
```

Comment:
isn't inner going to be links?  I'm not sure you'd want a link in a button.

Reply:
You are totally right.  I force-pushed a change to my branch to avoid this.